### PR TITLE
Display topic and exam for user

### PR DESCRIPTION
**Pull request by nguyenducanhst96**
image

### CODE REVIEW

**Comment by nguyenducanhst96** [comment]
https://dev.framgia.com/issues/184583

**Comment by nguyenducanhst96** [comment]
ready

File: app/views/reading/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,77 @@
+	<!-- Home -->
```
Comment (huongnguyenmta) [inline app/views/reading/index.html.erb]: xem lại indent file này nhé

**Comment by huongnguyenmta** [comment]
commented
Chị có comment nhỏ nhé ;)

**Comment by nguyenducanhst96** [comment]
ready

**Pull request merged**